packages/langium-cli/package.json

```diff
@@ -34,15 +34,15 @@
     "publish:latest": "npm publish --tag latest"
   },
   "dependencies": {
-    "colors": "^1.4.0",
     "commander": "^7.2.0",
     "fs-extra": "^9.1.0",
     "jsonschema": "^1.4.0",
     "langium": "~0.4.0",
     "lodash": "^4.17.21"
   },
   "devDependencies": {
-    "@types/fs-extra": "^9.0.13"
+    "@types/fs-extra": "^9.0.13",
+    "chalk": "^4.1.2"
```

Comment:
Why this move to `devDependencies`? I think it won't work because the code depends on it at runtime. Same for `generator-langium`.

Reply:
hopefully resolved - see [24ac604](https://github.com/langium/langium/pull/578/commits/24ac604d9bf5b97d71e4fabd00ca3af89e25c1f3)